roles/openshift_health_checker/openshift_checks/logging/podlogs.py

```diff
@@ -0,0 +1,135 @@
+        if not len(kibanna_pods):
+            return {"failed": True, "changed": False, "msg": ('No Kibanna pods were found to be' 
+                                                              'in the "Running" state.'
+                                                              'At least one Kibana pod is required' 
+                                                              'in order to perform this check.')}
+
+        self.logging_namespace = get_var(task_vars, "openshift_logging_namespace", default=self.logging_namespace)
+        pods, error = super(Podlogs, self).get_pods_for_component(
+            self.execute_module,
+            self.logging_namespace,
+            "es",
+            task_vars,
+        )
+
+        if error:
+            return {"failed": True, "changed": False, "msg": error}
+
+        es_pods = super(Podlogs, self).running_pods(pods)
+        if not len(es_pods):
+            return {"failed": True, "changed": False, "msg": ('No Elasticsearch pods were found to be'
+                                                              'in the "Running" state.'
+                                                              'At least one Kibana pod is required'
+                                                              'in order to perform this check.')}
+
+        try:
+            uid = self.curl_kibana_with_uuid(kibanna_pods[0], task_vars)
+            self.wait_until_cmd_or_err(es_pods[0], pod_es_host, pod_es_index, uid, pod_log_timeout, task_vars)
+        except OpenShiftCheckException as error:
+            return {"failed": True,  "msg": str(error)}
+        except ValueError as error:
+            return {"failed": True, "msg": "Missing key in json response from pod: {}".format(str(error))}
+
+    def wait_until_cmd_or_err(self, es_pod, es_host, es_index, uid, timeout_secs, task_vars):
+        """Wait a maximum of timeout_secs for the uuid to be found in Elasticsearch logs.
+        Since we are querying for a message with the the unique uuid from earlier,
+        there should only be a single match.
+        Raise an OpenShiftCheckException if the timeout is reached before a result is found."""
+
+        interval = 1
+
+        while timeout_secs > 0:
+            time.sleep(interval)
+            timeout_secs -= interval
+
+            total = self.query_es_from_es(es_pod, es_host, es_index, uid, task_vars)
+            if total == 1:
+                return
+
+        msg = "expecting match in Elasticsearch for message with uuid {}, but no matches were found after {}s."
+        raise OpenShiftCheckException(msg.format(uid, timeout_secs))
+
+    def curl_kibana_with_uuid(self, kibanna_pod, task_vars):
+        """curl Kibanna with a unique uuid to search for in Elasticsearch."""
+        uid = str(uuid4())
+        pod_name = kibanna_pod["metadata"]["name"]
+        exec_cmd = ("exec {pod_name} -c kibana -- curl --connect-timeout 1 -s "
+                    "http://localhost:5601/{uid}")
```

Comment:
> I'm surprised that apparently we create an entry with a GET request?!

This is how the script I am basing this off works: https://bugzilla.redhat.com/attachment.cgi?id=1258225

> Will this somehow pollute what users see, e.g. in the Web Console? Are they going to start seeing this mysterious UUIDs in their logs?

I believe this will happen, the 404 request is logged in kibana, and later aggregated into the elasticsearch logs. @sosiouxme wondering your thoughts on this as well